frontend/src/scenes/PreflightCheck/PreflightCheck.scss

```diff
@@ -0,0 +1,176 @@
+            border-radius: $radius;
+            border: 1px solid var(--border);
+
+            .validate-requirements-button-container {
+                display: flex;
+                justify-content: center;
+            }
+
+            .preflight-checks-list {
+                transition: all 1s ease-out;
+                overflow: hidden;
+            }
+
+            .preflight-check-summary {
+                display: flex;
+                padding: 12px;
+                border-bottom: 1px solid var(--border);
+                align-items: center;
+
+                .preflight-summary-icon-container {
+                    width: 40px;
+                    height: 40px;
+                    border-radius: 20px;
+                    display: flex;
+                    align-items: center;
+                    justify-content: center;
+                }
+
+                .preflight-summary-text-container {
+                    flex: 1;
+                    margin: 0 14px;
+
+                    p {
+                        margin-bottom: 0;
+                    }
+
+                    .check-summary-header {
+                        font-weight: 600;
+                        font-size: 14px;
+                    }
+
+                    .check-summary-description {
+                        font-size: 13px;
+                    }
+                }
+            }
+        }
+
+        .preflight-item {
+            display: flex;
+            padding: 12px;
+            align-items: center;
+            justify-content: center;
+            border-bottom: 1px solid var(--border);
+
+            p {
+                margin-bottom: 0;
+                font-size: 14px;
+            }
+
+            .icon-container {
+                display: flex;
+            }
+
+            .central-text-container {
+                flex: 1;
+                margin: 0 14px;
+
+                .check-name {
+                    font-weight: 600;
+                }
+            }
+
+            svg {
+                font-size: 20px;
+            }
+        }
+    }
+
+    .cannot-continue {
+        background-color: var(--bg-side);
+        border: 2px solid $light-grey;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        p {
+            font-size: 14px;
+            margin-bottom: 0;
+        }
+    }
+}
+
+.loading {
+    background-color: #f6f8ff;
+
+    svg,
+    .status-text {
+        color: var(--primary);
+    }
+}
+
+.validated {
+    background-color: #f8fcf8;
+
+    svg,
+    .status-text {
+        color: var(--success);
+    }
+}
+
+.warning {
+    background-color: #f7a5011a;
+
+    svg,
+    .status-text {
+        color: var(--warning);
+    }
+}
+
+.optional {
+    background-color: #fafaf9;
+
+    svg,
+    .status-text {
+        color: var(--border-dark);
+    }
+}
+
+.error {
+    background-color: #feefea;
+
+    svg,
+    .status-text {
+        color: var(--danger);
+    }
+}
```

Comment:
This is dangerous! Those classes will apply globally and their names are super generic, so it's likely to be an issue.

Reply:
I really like the [BEM](https://css-tricks.com/bem-101/) convention for naming classes which basically eliminates such collisions, but it's not a hard rule at the moment.

Downsides:
1. It can be verbose

Upsides:
1. It makes you think about the structure of your component more
2. It makes that structure much more obvious to readers of the code
3. It's collision-proof

Reply:
Just tried out using BEM, I think I liked it. 

It's a bit awkward with how it interacts with sass. I felt like I was duplicated a good amount when nesting something and having the Block in the name, but overall I like that it gives some convention to css naming which otherwise feels like a free for all.